certified-connectors/CiviQ/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2868 @@
+    "version": "1.2",
+    "contact": {
+      "name": "Mark O'Toole",
+      "url": "https://civiq.freshdesk.com",
+      "email": "support@civiq.eu"
+    }
+  },
+  "host": "consult.civiq.eu",
+  "basePath": "/en/api/1.1/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/consultations.json": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Consultation data",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "count": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Consultations count"
+                },
+                "consultations": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "nid": {
+                        "type": "string",
+                        "description": "Consultation nid"
+                      },
+                      "type": {
+                        "type": "string",
+                        "description": "Consultation type"
+                      },
+                      "created": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "Consultation created data"
+                      },
+                      "preset": {
+                        "type": "string",
+                        "description": "Consultation preset"
+                      },
+                      "access": {
+                        "type": "string",
+                        "description": "Consultation access"
+                      },
+                      "url": {
+                        "type": "string",
+                        "description": "Consultation url"
+                      },
+                      "status": {
+                        "type": "string",
+                        "description": "Consultation status"
+                      },
+                      "title": {
+                        "type": "string",
+                        "description": "Consultation title"
+                      },
+                      "image": {
+                        "type": "object",
+                        "properties": {
+                          "uuid": {
+                            "type": "string",
+                            "description": "Image uuid"
+                          },
+                          "name": {
+                            "type": "string",
+                            "description": "Iamge name"
```

Comment:
@RicardoMartins1993 all files should be free of grammatical errors and typos. please change "Iamge" to "Image". Please fix in all the swagger files.

Reply:
@Amjed-Ayoub  Done